stdlib/2and3/distutils/cmd.pyi

```diff
@@ -13,3 +13,28 @@ class Command:
     def finalize_options(self) -> None: ...
     @abstractmethod
     def run(self) -> None: ...
+
+    def announce(self, msg: str, level: int = ...) -> None: ...
+    def debug_print(self, msg: str) -> None: ...
+
+    def ensure_string(self, option: str, default: Optional[str]) -> None: ...
+    def ensure_string_list(self, option: str) -> None: ...
+    def ensure_filename(self, option: str) -> None: ...
+    def ensure_dirname(self, option: str) -> None: ...
+
+    def get_command_name(self) -> str: ...
+    def set_undefined_options(self, src_cmd: str, *option_pairs: Sequence[Tuple[str, str]]) -> None: ...
+    def get_finalized_command(self, command: str, create: int = ...) -> Command: ...
+    def reinitialize_command(self, command: Union[Command, str], reinit_subcommands: int = ...) -> Command: ...
+    def run_command(self, command: str) -> None: ...
+    def get_sub_commands(self) -> List[str]: ...
+
+    def warn(self, msg: str) -> None: ...
+    def execute(self, func: Callable[..., Any], args: List[Any], msg: Optional[str] = ..., level: int = ...) -> None: ...
```

Comment:
Can the List be a Sequence or even Iterable instead?

Reply:
The only constraint is probably the unpacking, so I would guess `Sequence[Any]`, but I'm not completely sure. The underlying code called is this: (Command.execute calls util.execute)
https://github.com/python/cpython/blob/master/Lib/distutils/util.py#L270

Reply:
Unpacking works with any Iterable.